src/audio/drc/drc_math_hifi3.c

```diff
@@ -62,35 +50,49 @@ static inline ae_f32 log10_fixed(ae_f32 x)
 	 * fpminimax(log10(x), 5, [|SG...|], [1/2;sqrt(2)/2], absolute);
 	 * max err ~= 6.088e-8
 	 */
-	int32_t e;
-	int32_t lshift;
 	ae_f32 exp; /* Q7.25 */
 	ae_f32 acc; /* Q6.26 */
 	ae_f32 tmp; /* Q6.26 */
+	ae_f64 tmp64;
```

Comment:
So, there's overhead in inline functions, maybe it's from 26 as literal instead of variable? You could comment that the instructions normalize the value after the function was removed.

Reply:
I tested in xtensa test bench, it really costs more cycles when using the function wrapper than instructions.